Tasks/Common/kubernetes-common-v2/image-metadata-helper.ts

```diff
@@ -0,0 +1,246 @@
+import * as tl from "azure-pipelines-task-lib/task";
```

Comment:
Add UTs for exported and public methods.

Reply:
Added ToDo for the same